src/SALib/util/__init__.py

```diff
@@ -99,71 +100,96 @@ def unscale_samples(params, bounds):
 def nonuniform_scale_samples(params, bounds, dists):
     """Rescale samples in 0-to-1 range to other distributions
 
-    Arguments
-    ---------
+    Parameters
+    ----------
     problem : dict
         problem definition including bounds
     params : numpy.ndarray
         numpy array of dimensions num_params-by-N,
         where N is the number of samples
-    dists : list
-        list of distributions, one for each parameter
+    dists : list or str
+        list - a list of distributions, one for each parameter
             unif: uniform with lower and upper bounds
             triang: triangular with width (scale) and location of peak
                     location of peak is in percentage of width
                     lower bound assumed to be zero
-            norm: normal distribution with mean and standard deviation
+            norm: normal distribution with mean and standard deviation or
             lognorm: lognormal with ln-space mean and standard deviation
+        str - distribution applied to all parameters
+            multi_norm: Correlated distribution using the Cholesky 
+            decomposition approach.
+            In this case `bounds` specifies mean (first column) and desired
+            covariance (other columns).
     """
     b = np.array(bounds)
 
     # initializing matrix for converted values
     conv_params = np.empty_like(params)
+    conv_len = conv_params.shape[1]
+
+    is_str = isinstance(dists, str)
+    if is_str and (dists == 'multi_norm'):
```

Comment:
Would 'cholesky' perhaps be more informative?

Reply:
Cholesky is just a decomposition method, not a distribution composition. This label may be not so associative. But, if you have strong wish - I don't mind.

Reply:
I don't have a strong desire to change the name, no. I only wish to convey how the "multi_norm" is generated.

Perhaps we could provide a thin wrapper around the [numpy function](https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.multivariate_normal.html) which would give us access to other methods at the same time...

What do you think?

Reply:
Why not. However multivariate_normal from numpy generates a sample and is not able to correlate already existing sample.

Reply:
Oh yes, sorry I obviously didn't look very closely.